ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot

```diff
@@ -11,29 +13,31 @@ ${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
 Verify TrustyAI Operator Installation
     [Documentation]    Verifies that the TrustyAI operator has been
     ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
-    [Tags]    Smoke
-    ...       Tier1    ODS-2481
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Deployment
+    [Tags]    Tier1    Sanity
+    ...       ODS-2481    robot:recursive-continue-on-failure
+    Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Operator Deployment
     Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI ReplicaSets Info
     Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI Container Names
 
 
 *** Keywords ***
-Verify trustyai-service-operator-controller-manager Deployment
-    [Documentation]    Verifies the  deployment of the trustyai operator in the namespace
-    Wait For Pods To Be Ready   label_selector=app.kubernetes.io/created-by=trustyai-service-operator
-    ...    namespace=${APPLICATIONS_NAMESPACE}    exp_replicas=1
+Verify TrustyAI Operator Deployment
+    [Documentation]    Verifies the  deployment of the trustyai operator in the Applications namespace
+    Wait For Pods Number  1
```

Comment:
why this change?

Reply:
The Operator name was changed from now and when the PR was written and it was failing. Fixed it now. Works fine.

Reply:
ok but why changing the kw?

Reply:
kW title has been changed.